src/assets/src/components/queueManager.tsx

```diff
@@ -323,19 +325,18 @@ export function QueueManagerPage(props: PageProps<QueueManagerPageParams>) {
 
     // Render
     const isChanging = removeMeetingLoading || addMeetingLoading || setStatusLoading || changeAssigneeLoading || startMeetingLoading;
-    const errorSources = [
+    const globalErrorSources = [
         {source: 'Access Denied', error: authError},
         {source: 'Queue Connection', error: queueWebSocketError},
         {source: 'User Connection', error: userWebSocketError},
         {source: 'Remove Meeting', error: removeMeetingError},
-        {source: 'Add Meeting', error: addMeetingError},
         {source: 'Queue Status', error: setStatusError},
-        {source: 'Assignee', error: changeAssigneeError},
```

Comment:
Looks like I accidentally deleted this when I was playing around with it. Going to add it back.

Reply:
Done (https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/308/commits/6d1a3eb471772492095dd404cb5ec8a054ec8b88)